selfdrive/test/process_replay/test_processes.py

```diff
@@ -127,6 +151,7 @@ def format_diff(results, ref_commit):
                       help="Updates reference logs using current commit")
   parser.add_argument("--upload-only", action="store_true",
                       help="Skips testing processes and uploads logs from previous test run")
+  parser.add_argument("--jobs", action="store_true")
```

Comment:
Maybe make an int so you can specify different number of jobs?

```suggestion
  parser.add_argument("-j", "--jobs", type=int, default=1)
```

Reply:
Depends on the use case. Which interface is preferred by pipeline?

Reply:
Doesn't make sense to have an argument named jobs that doesn't take a number of jobs. If you're going to hard code, you should probably at least use `os.cpu_count()`.

Reply:
yeah, should be an int. 1 is fine as the default, same as make